internal/platform/database/tenantquery/repository_test.go

```
@@ -0,0 +1,231 @@
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+)
+
+func TestListAccessibleReturnsAssignedPropertyTenants(t *testing.T) {
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("sqlmock.New: %v", err)
+	}
+	defer db.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `db.Close` is not checked

(errcheck)